packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -0,0 +1,112 @@
+
+You can investigate high usage of a rate limit or a rate limit violation. After you know the issue, you can take the appropriate corrective action. For example, if the issue is high traffic, you may want to increase the rate limits. In cases where the issue is a rogue app or bad script, you can disable the app, or block that traffic. Whatever the issue is, you can remediate the issue through self-service. In cases where a higher rate limit is required, you can create a support case, by sending a request to the correct team for a rate limit increase.
+
+## Access the dashboard
+
+When a rate limit violation occurs, you're notified in the following ways and can access the rate limits dashboard from the link in the notification:
+
+* You receive a rate limit email that informs you that you've hit a rate limit for an endpoint. Click **View utilization history** in the email to navigate to the Rate Limits Dashboard and System Log for that particular endpoint.
+
+![Access the dashboard by email notification](/img/RLDashboard1.png "Displays the email notification of a rate limit violation with the link to the endpoint's rate limit dashboard")
+
+* You've hit a rate limit when you're in the Admin Console, and a banner appears to inform you. Click **utilization history** in the banner to open the Rate Limits Dashboard for that particular endpoint or **syslog** to open the System Log.
+
+![Access the dashboard by link in the banner](/img/RLDashboard2.png "Displays the banner in the dashboard that notifies you of a rate limit violation with the link to the endpoint's rate limit dashboard")
+
+* You've hit a rate limit warning or violation, and a rate limit event (violation or warning) is created in the System Log from where you can click **View Utilization** under **Event info** to go to the rate limits dashboard. All the rate limit events in the System Log (violations and warnings) have a link to the rate limit dashboard.
+
+![Access the dashboard by link in the System Log](/img/RLDashboard3.png "Displays the event in the System Log that notifies you of a rate limit violation with the link to the endpoint's rate limit dashboard")
+
+## Key components of the dashboard
+
+The rate limit dashboard provides data usage information for the endpoint based on a set of statistics and shows multiple views of the data usage. The dashboard helps you to investigate the rate limit or violation issue and to prevent subsequent rate limit issues from occurring. To view the most recent data, refresh your screen. There may be a slight delay of two to four minutes in obtaining new data.
```

Comment:
```suggestion
The rate limit dashboard provides data use information for the endpoint based on a set of statistics and shows multiple views of the data use. The dashboard helps you to investigate the rate limit or violation issue and to prevent subsequent rate limit issues from occurring. To view the most recent data, refresh your screen. There may be a slight delay of two to four minutes in obtaining new data.
```